tests/services/test_players.py

```
@@ -0,0 +1,156 @@
+    "player_id",
+    [
+        "3373",  # Ronaldinho
+        "8198",  # Cristiano Ronaldo
+        "28003",  # Lionel Messi
+        "68290",  # Neymar
+    ],
+)
+def test_players_profile(player_id: str):
+    tfmkt = TransfermarktPlayerProfile(player_id=player_id)
+    response = PlayerProfile.model_validate(tfmkt.get_player_profile())
+    expected = PlayerProfile.model_validate(load_test_data(f"players/profile/{player_id}"))
```

Ruff (0.8.2)

Undefined name `load_test_data`

(F821)